pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -993,6 +993,9 @@ func GetOpenshiftBootstrapClusterRoleBindings() []rbac.ClusterRoleBinding {
 		newOriginClusterBinding(BuildStrategyJenkinsPipelineRoleBindingName, BuildStrategyJenkinsPipelineRoleName).
 			Groups(AuthenticatedGroup).
 			BindingOrDie(),
+		rbac.NewClusterBinding(NodeBootstrapRoleName).
```

Comment:
it certainly needs a separating comment, not be grouped with the build strategies roles